src/App.tsx

```diff
@@ -15,47 +16,140 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
```

Comment:
```suggestion
  None,
```

fix other properties in the enum as well

Reply:
Enums were originally in the uppercase from the last task, but thanks for the feedback.